.env.example

```
@@ -70,3 +70,10 @@ AWS_BUCKET=
 AWS_USE_PATH_STYLE_ENDPOINT=false
 
 VITE_APP_NAME="${APP_NAME}"
+
+# ============================================================
+# Google Calendar Integration
+# ============================================================
+GOOGLE_CALENDAR_SERVICE_ACCOUNT_JSON=storage/app/google-calendar-service-account.json
+GOOGLE_CALENDAR_ID=your_calendar_id_here
+GOOGLE_CALENDAR_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_CALENDAR_ENABLED key should go before the GOOGLE_CALENDAR_ID key

(UnorderedKey)